polkadot-wiki/sidebars.js

```diff
@@ -26,84 +26,108 @@ module.exports = {
       type: 'category',
       label: 'Learn',
       items: [
-        'learn/learn-launch',
         {
           type: 'category',
           label: 'Basics',
-          items: [
-            'learn/learn-architecture',
-            'learn/learn-accounts',
-            'learn/learn-account-generation',
-            'learn/learn-account-restore',
-            'learn/learn-polkadotjs',
-            'learn/learn-assets',
-            'learn/learn-nft',
-            'learn/learn-DOT',
-            'learn/learn-security',
-            'learn/learn-consensus',
-            'learn/learn-nominator',
-            'learn/learn-staking-faq',
-            'learn/learn-validator',
-            'learn/learn-collator',
-            'learn/learn-governance',
-            'learn/learn-identity',
-            'learn/learn-balance-transfers',
-            'learn/learn-transaction-fees',
-            'learn/learn-polkadot-host',
-            'learn/learn-treasury',
-            'learn/learn-registrar',
-            'learn/learn-runtime-upgrades',
-          ],
-        },
-        {
-          type: 'category',
-          label: 'Paraobjects',
           items: [
             {
               type: 'category',
-              label: 'Parachains',
+              label: 'Accounts',
               items: [
-                'learn/learn-parachains',
-                'learn/learn-common-goods',
-                'learn/learn-auction',
-                'learn/learn-crowdloans',
-                'learn/learn-teleport',
+                  'learn/learn-accounts',
+                  'learn/learn-account-generation',
+                  'learn/learn-account-restore',
+                  'learn/learn-balance-transfers',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Assets',
```

Comment:
Same title in the dropdown Assets > Assets 

```suggestion
              label: 'Tokens and Assets',
```